docs/ops/arithmetic/Atan_1.md

```diff
@@ -22,12 +28,6 @@
 
 * *T*: any numeric type.
 
-*atan* does the following with the input tensor *a*:
-
-\f[
-a_{i} = atan(a_{i})
-\f]
-
 **Examples**
 
 *Example 1*
```

Comment:
When there is only one example, what we are doing is cleaning up \*Example 1\* and leave just \*\*Examples\*\*

Reply:
updated in doc.